### ISSUE TITLE
no_unecessary_fat_arrows false positive for class methods inside function

### ISSUE THREAD

**Issue by jonahkagan** [clutchski/coffeelint#213]
If you create a class with a class method inside a function, you get a warning from no_unecessary_fat_arrows. For example:

``` coffee
outer_function = ->
  class Yikes
    @class_method: -> 'no errors here'
```

Gives this error:

```
#1: Used `this` in a function without a fat arrow.
```

I can look into fixing it but don't have the time this second.
